server/server.go

```diff
@@ -46,10 +60,29 @@ func main() {
 
 	r := internal.NewServer(storageClient, containerName)
 
-	udpPort := 5688
-	dTLSPort := 5689
-	udpAddr := fmt.Sprintf("%s:%d", *address, udpPort)
-	dtlsAddr := fmt.Sprintf("%s:%d", *address, dTLSPort)
+	go func() {
```

Comment:
Refactor this to reuse the code that launches the server and not duplicate it.

Reply:
HI @coderbyheart , thanks for being with me. That indeed was another learning point for me towards maintaining clean code. Request some of your busy time to please review this change. Thanks again !!